cmd/cluster-agent/api/server.go

```diff
@@ -148,6 +148,7 @@ func isExternalPath(path string) bool {
 		strings.HasPrefix(path, "/api/v1/tags/pod/") && (len(strings.Split(path, "/")) == 6 || len(strings.Split(path, "/")) == 8) ||
 		strings.HasPrefix(path, "/api/v1/tags/node/") && len(strings.Split(path, "/")) == 6 ||
 		strings.HasPrefix(path, "/api/v1/tags/namespace/") && len(strings.Split(path, "/")) == 6 ||
+		strings.HasPrefix(path, "/api/v1/annotations/node/") && len(strings.Split(path, "/")) == 6 ||
```

Comment:
can we build a more generic endpoint like
`/api/v1/node` and have a structured payload like
```json
{  "host_aliases": ["my-host"] }
```
I have checked the rest of the PR yet, but IMO we should only return the useful information and not send all the annotations, because they can be huge.

Reply:
That’s definitively possible but it will change the way the feature is working.

With the current state of the PR, the `kubernetes_node_annotations_as_host_aliases` parameter is set at the node agent level and it’s possible (although I can’t see the use case) to have different values for serval node agents.

With this proposition, the parameter will be set at the cluster agent level and will be global for the whole cluster.
That means that the host aliases reported by node agents will depend on the parameters of the DCA.

Reply:
I’ve implemented your suggestion:
A new parameter `kubernetes_node_annotations` parameter is used by the DCA to restrict the amount of annotations returned back to the node agents.